experimental/dds/tree2/src/test/shared-tree-core/sharedTreeCore.spec.ts

```diff
@@ -281,6 +281,7 @@ describe("SharedTreeCore", () => {
 		} satisfies InitializeAndSchematizeConfiguration;
 
 		const view1 = tree1.schematizeInternal(config);
+		containerRuntimeFactory.processAllMessages();
```

Comment:
This line is required to make this test actually test what it claims; however, without this line, it fails in delta application. This is a bug and should be investigated.